rskj-core/src/main/java/co/rsk/scoring/PeerScoringSummaryService.java

```diff
@@ -0,0 +1,49 @@
+package co.rsk.scoring;
+
+import co.rsk.config.InternalService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class PeerScoringSummaryService implements InternalService {
+
+    private static final Logger logger = LoggerFactory.getLogger("peerScoring");
+
+    private final long REPORT_TIME;
+    private final PeerScoringManager peerScoringManager;
+
+    public PeerScoringSummaryService(long peerScoringStatsReportTime, PeerScoringManager peerScoringManager) {
+        this.REPORT_TIME = peerScoringStatsReportTime;
+        this.peerScoringManager = peerScoringManager;
+    }
+
+    @Override
+    public void start() {
+        logger.debug("starting peer scoring summary service");
+        try {
+            ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
+            executorService.scheduleAtFixedRate(() -> this.report(peerScoringManager.getPeersInformation()),
+                    0,
+                    REPORT_TIME,
+                    TimeUnit.MILLISECONDS
+            );
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
logger.error(...)

Reply:
i think this one it's too light to be considered as an error, what do you expect from an error?